docs/source/main_classes/trainer.rst

```diff
@@ -119,6 +119,60 @@ TFTrainingArguments
     :members:
 
 
+Logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default :class:`~transformers.Trainer` will use ``logging.INFO`` for the main node and ``logging.WARNING`` for the
+replica nodes if any.
+
+These defaults can be overridden to use any of the 5 ``logging`` levels with :class:`~transformers.TrainingArguments`'s
+arguments:
+
+- ``log_level`` - for the main node
+- ``log_level_replica`` - for the replica nodes
+
+Further, if :class:`~transformers.TrainingArguments`'s ``log_on_each_node`` is set to ``True`` all nodes will use the
+log level settings for the main node.
```

Comment:
```suggestion
Further, if :class:`~transformers.TrainingArguments`'s ``log_on_each_node`` is set to ``True`` all nodes will use the
log level settings for the main process.
```
This is actually the default, so maybe formulate this in the other direction? 

Further, if :class:`~transformers.TrainingArguments`'s ``log_on_each_node`` is set to ``False`` only the main node will use the log level settings for its main process, all other nodes will use the log level settings for replicas.

Reply:
excellent. Thank you, Sylvain! 

I extended the examples to include this too, so it's even more clear now.